app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ProfilesController < ApplicationController
+  before_action :require_authentication
+
+  def show; end
+
+  def update
+    return unless current_user?
+
+    @user = current_user
+    if @user.update(profile_params)
+      flash[:success] = 'Your profile has been updated'
+      render json: @user.to_json
+    else
+      render json: { errors: @user.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
+
+  def destroy; end
```

Comment:
This endpoint is not being used? Also, according to the process we'll be manually deleting users for now (when requested).

Reply:
Yup youre right. Removed